cli/src/commands/schema/edit.ts

```diff
@@ -453,6 +457,28 @@ Beware that this can lead to ${chalk.bold(
             return true;
           }
         },
+        {
+          name: 'vectorDimension',
+          message: 'Vector Dimension. Only for columns that are vectors',
+          validate(
+            value: string | undefined,
+            state: ColumnEditState,
+            item: { value: string | undefined },
+            index: number
+          ) {
+            if (!isColumnAdded && value !== state.initial.vectorDimension) {
+              return `Cannot change the vector dimension of existing vector columns`;
+            }
+            if (state.values.type === 'vector') {
+              if (!value) {
+                return 'The vectorDimension field must be filled the columns of type `vector`';
```

Comment:
Is a word missing here?

```suggestion
                return 'The vectorDimension field must be filled for the columns of type `vector`';
```